web/src/portico/team.ts

```diff
@@ -30,6 +43,17 @@ const repo_name_to_tab_name: Record<RepositoryName, string> = {
     "zulip-terminal": "terminal",
     "zulip-ios-legacy": "",
     "zulip-android": "",
```

Comment:
Should these two now be mapped to `mobile`?

Reply:
Though really, I think this data structure should now be computable from `tab_name_to_repo_list`; we should just do that computation (or the reverse one) rather than duplicating the mappings.

Reply:
Dropped this Data Structure